build/build-allocation-images/go/gen.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+export GO111MODULE=on
+
+mkdir -p /go/src/
+cp -r /go/src/agones.dev/agones/vendor/* /go/src/
+
+cd /go/src/agones.dev/agones
+go install -mod=vendor github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway
+go install -mod=vendor github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger
+
+outputpath=./allocation/go
+protopath=cmd/allocator/v1alpha1
```

Comment:
I wouldn't have expected the proto source to be under `cmd`. The `cmd` directory should be for main program entry points, and the proto definition is something that generates shared code. 

Now that we have multiple proto files, it seems like it would be nice to create a folder to hold them (e.g. `proto/`) at the root level. The proto file isn't being added in the change, so I don't want to block you from making progress, but I do think we should find it a better home.

Reply:
I am addressing this comment in: https://github.com/googleforgames/agones/pull/1041
I will update the protopath reference after the PR is merged.

Reply:
Done.